README.md

```diff
@@ -111,30 +111,52 @@ T::Coerce[Params].new.from({id: '1'})
 ```
 More examples: [nested params](https://github.com/chanzuckerberg/sorbet-coerce/blob/a56c0c6a363bb49b11e77ac57893afc3d54c6b8c/spec/nested_spec.rb#L18-L26)
 
-## Coercion Error
+## Errors
+- `T::Coerce::CoercionError`: It fails to convert a value into the specified type (i.e. `'bad string args' to Integer`).
```

Comment:
```suggestion
- `T::Coerce::CoercionError`: It fails to convert a value into the specified type (i.e. `'bad string args' to Integer`). Can be configured to use `nil` instead of raise error. See below.
```